planet4-gutenberg-blocks.php

```diff
@@ -47,6 +47,9 @@
 if ( ! defined( 'P4GBKS_PLUGIN_DIR' ) ) {
 	define( 'P4GBKS_PLUGIN_DIR', WP_PLUGIN_DIR . '/' . P4GBKS_PLUGIN_DIRNAME );
 }
+if ( ! defined( 'P4GBKS_PLUGIN_DIR_PATH' ) ) {
+	define( 'P4GBKS_PLUGIN_DIR_PATH', plugin_dir_path( __FILE__ ) );
```

Comment:
Hey @sagarsdeshmukh isn't this the same as the above constant `P4GBKS_PLUGIN_DIR`? They are in my local environment, except for a trailing slash in the new one.

Reply:
Indeed, no need of new P4GBKS_PLUGIN_DIR_PATH constant.